.github/workflows/canary-aarch64-linux.yml

```diff
@@ -0,0 +1,81 @@
+name: Canary-Aarch64-Linux
+
+on:
+  schedule:
+    - cron: '0 18 * * *'
+  pull_request:
```

Comment:
We don't want to run on pull request, nightly only

Reply:
I removed the pull_request trigger